README.md

```diff
@@ -2,8 +2,38 @@
 
 This is the source code for the Scala.js website at
 [http://www.scala-js.org/](http://www.scala-js.org/).
-It is built with [Jekyll](http://jekyllrb.com/).
-Pull requests welcome!
 
-Looking for the source code of Scala.js itself?
-It is [over there](https://github.com/scala-js/scala-js).
+If you intended to look at the Scala.js source code itself,
+you can find it here: [github.com/scala-js/scala-js](https://github.com/scala-js/scala-js).
+
+# Contributing
+
+The key to contributing is being able to edit and
+preview your content. [Your pull requests are welcome](https://github.com/scala-js/scala-js-website/compare)!
+
+## Set up
+As this website is built with [Jekyll](http://jekyllrb.com/),
+we will need to set up some Ruby tooling.
+
+First, install RVM (Ruby Version Manager): https://rvm.io/rvm/install
+Then run the following commands:
+```bash
+$ source ~/.rvm/scripts/rvm
+# Use Ruby 2.2.3
+$ rvm use 2.2.3 --install --fuzzy
```

Comment:
What is `--fuzzy`? I don't find any real documentation on what it does, and I don't remember using that flag ever. Can we leave it out?